packages/frontend/src/utils/wallet.js

```diff
@@ -142,8 +142,18 @@ class Wallet {
         this.accountId = localStorage.getItem(KEY_ACTIVE_ACCOUNT_ID) || '';
     }
 
+    removeWalletAccount(accountId) {
+        let walletAccounts = this.getAccountsLocalStorage();
+        delete walletAccounts[accountId];
+        setWalletAccounts(KEY_WALLET_ACCOUNTS, walletAccounts);
+        this.keyStore.removeKey(NETWORK_ID, accountId);
```

Comment:
This method is `async` - we need to either `await` it or add a .catch() to it that swallows the error or logs the error, or any errors calling `removeKey()` will become unhandled promise rejections